lib/loggerator/loggerator.rb

```diff
@@ -1,17 +1,30 @@
-require_relative 'request_store'
-require_relative 'middleware'
+require "loggerator/log"
+require "loggerator/configuration"
+require "loggerator/request_store"
+require "loggerator/middleware"
 
 module Loggerator
-
   def self.included(mod)
     mod.extend self
   end
 
+  def self.config
+    @@config ||= Configuration.new
```

Comment:
I _thought_ it needed to be `@@` because it's in a `self` block. However, I'm down to try it out.